### PR TITLE
Added fork and clone instructions with images

### PR DESCRIPTION
**Pull request by jchansen925**
This is the first set of documentation I have worked on. Just want to make sure I am heading in the right direction.

### CODE REVIEW

File: docs/ForkClone.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+You will now have a forked version of the original repo that is up to date with the master branch of the original repo.
+
+![](ForkedBranch.png)
+
+## Steps to clone the repo:
+
+### Step 1
+
+Clone this repo to your local machine using (https://github.com/gbowne1/codebooker.git)
```
Comment (LOGESH-B) [inline docs/ForkClone.md:40]: I think, on this place we need to clone the forked repo.Not the main repo

**Review by gbowne1 — APPROVED** [review]
This documenation looks great as initial work for the development documentation which should eventually be pushed to our GitHub wiki.

Great to see included pictures.

File: docs/ForkClone.md (hunk 1 above)
Comment (gbowne1) [reply docs/ForkClone.md:40]: One could definitely either clone or fork.  They are similar but do different things: 

<https://stackoverflow.com/questions/7057194/what-is-the-difference-between-forking-and-cloning-on-github>

Comment (LOGESH-B) [reply docs/ForkClone.md:40]: Yeah, But I thing these steps are for cloning the forked project ((i.e) the continuation of the forked project🤷‍♂️

**Review by LOGESH-B — APPROVED** [review]
Thanks for your Contibution @jchansen925 
Approving for merge✌️

**Comment by jchansen925** [comment]
Hi, I have made some updates to the Fork and Clone documentation, and have
some today for more doc work. I am working on the pull request doc now. One
question I have is where to get more information for documenting the
following topics:

Client Server Dependencies,
Create Database,
Run Lint Prettier,
Run Tests,
Set Up in Editors.

Also, for Install, can I just take that information from the DEVSETUP.md
file, or is there more detailed information needed.
And for React Instructions, should that information also be taken from the
"Learn More" section of the same Dev Setup doc?

Thanks,

John

**Comment by gbowne1** [comment]
@jchansen925 

Here are some basic instructions for:

## Installing Client & Server Dependencies

 The client dependencies are in the project root, so in ~/codebooker/ you would run `npm install`
 The server dependencies are in the /src/server folder so change to /src/server and run `npm install`
 
 A person could use yarn or pnpm if they wanted to as yarn is compatible with npm. Yarn will make a yarn.lock  I have no experience yet with pnpm, but assume its also compatible.
 
## Create the MongoDB database

If you have installed MongoDB locally, you can use a MongoDB shell such as mongo or mongosh to create the database.  If you are using MongoDB Compass or Atlas, there are specific instructions for that.

To create the 'codebooker' database in mongo or mongosh run the command `use codebooker`.

## Running ESLint

Before you commit, push and PR your code you should run ESLint.  To do this, run the command `npx eslint --fix .` inside the project root directory.  This will expose any problems with the code as warnings.

## Running Prettier

Before you commit, push and PR your code you should run Prettier. 

To do this, run the command `npx prettier --write .` inside the project root directory.  This will clean up the code so that it is uniform with the codebase.

## Running Tests

We do not have any tests written yet.  We will be using Jest and the React Testing Library `@testing-library`.  So links to Jest and React Testing Library documentation will work just fine

Setting up this project in different editors depends on which editors or IDE's are used, so I would not expect this documentation to be done right away.  This would be good for beginners.

If you want to modify or remove the DEVSETUP.md that is fine. That is documentation that I copied from a previous version of React (16) and the links in that document are all dead, although a portion of the DEVSETUP.md is correct, plenty has changed since 16 came out, especially now that they have abandoned Create React App.  The new React documentation is here: https://react.dev/  I made DEVSETUP.md as developer getting started documentation particularly aimed at beginners who have never used React before.

**Comment by jchansen925** [comment]
Ok, thanks, this is a great start for the rest of these documents,

John

**Pull request merged**